front-end/src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import Activities from './pages/Activities'
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Activities />, document.getElementById('root'));
```

Comment:
this means you are no longer using App as your root and so no other items are possible.  you should instead import Activities in to App and display the Activities as a child component of App

Reply:
Got it.